source/index-management.rst

```diff
@@ -782,6 +782,15 @@ Common API Components
      * Optionally specifies the wildcard projection of a wildcard index.
      */
     wildcardProjection: Document;
+
+    /**
+     * Optionally specifies that the index should exist on the target collection but should not be used by the query
+     * planner when executing operations.
+     *
+     * This option is only supported by servers >= 4.4. For servers < 3.4, drivers MUST raise an error if the caller
```

Comment:
Why must drivers raise for this one, when they don't for any of the other options above?

Reply:
I think it's consistent to validate these options like we do for other command options in the CRUD spec. I filed https://jira.mongodb.org/browse/DRIVERS-1220 to address this for the others too. I'm also fine with removing this clause right now and deferring to the decision made for DRIVERS-1220 if you think that'd be better.

Reply:
I'd prefer to be consistent with the other options for this command.

Reply:
Removed the constraint for raising an error.